packages/framework/src/Form/Admin/Mail/MailTemplateFormType.php

```diff
@@ -49,18 +50,18 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     ])
                     ->addModelTransformer(new EmptyWysiwygTransformer())
             )
-            ->add('attachment', FileUploadType::class, [
+            ->add('attachments', FileUploadType::class, [
                 'required' => false,
-                'multiple' => false,
                 'file_constraints' => [
                     new Constraints\File([
                         'maxSize' => '2M',
```

Comment:
Shouldn't be here constraint for allowed file types ? (`txt`, `xml` etc.)

Reply:
There are no constraints defined sadly.